### DISCUSSION TITLE
New model: CrisperWhisper

### DISCUSSION THREAD

**Discussion by imsakg** [SYSTRAN/faster-whisper#1163] [Ideas]
I would like to bring attention to the recent release of the new Whisper model, which has demonstrated significant performance on the [OpenASR leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard). This advancement in ASR technology is noteworthy and could have implications for faster-whisper.

References:
	- [CrisperWhisper](https://huggingface.co/nyrahealth/CrisperWhisper)
	- [OpenASR Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard)

**Comment by Purfview** [comment]
>This advancement in ASR technology is noteworthy and could have implications for faster-whisper.

Personally I didn't encountered advancement in any finetuned model.

This one I remember I canceled after the first few segments it outputted, with a thought - "Oh, this not gonna fly."

**Reply by Purfview** [reply]
Tried it again, on English audio. Bad punctuation, bad segmentation, bad chunking, weird hallucinations with initial prompt, maybe it's not "compatible" with long form transcriptions...
